### PR TITLE
Simplify nested templates with isolated context.

### PR DESCRIPTION
**Pull request by jmcarp**
This patch replaces the `in_context` tag with an arguably simpler
`render_nested` tag. The proposed tag renders an arbitrary template with
an arbitrary context, in isolation from the parent context. This is same
use case `in_context` is currently supporting, but with a bit less code.

Note: this also doesn't correspond to an issue--it's a thought I had while getting familiar with this repo and thinking about https://github.com/18F/atf-eregs/issues/207. Just a suggestion I wanted to get down before moving on.

@cmc333333

### CODE REVIEW

File: regulations/templates/regulations/macros/nested.html [hunk 1]
```diff
@@ -0,0 +1 @@
+{% extends template %}
```
Comment (cmc333333) [inline regulations/templates/regulations/macros/nested.html]: The existence of this template feels a little clunky -- if someone's trying to follow the code, they'd have to jump to an effectively empty file. Would it make sense to just render the template in `render_nested.py`?

One-liner:
```python
return loader.get_template(template).render(Context(context))
```

**Comment by cmc333333** [comment]
Very nice. Sometimes you go off the rails.

**Comment by cmc333333** [comment]
Bump

**Comment by jmcarp** [comment]
Patch updated.

**Comment by cmc333333** [comment]
Excellent

**Pull request merged**